runtime-light/tl/tl-functions.cpp

```diff
@@ -0,0 +1,48 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/tl/tl-functions.h"
+
+#include <cstdint>
+#include <string_view>
+
+#include "runtime-light/tl/tl-core.h"
+
+namespace {
+
+// magic + flags + image_id + job_id + minimum string size length
+constexpr auto K2_INVOKE_JW_MIN_SIZE = 2 * sizeof(uint32_t) + sizeof(uint64_t) + sizeof(int64_t) + tl::SMALL_STRING_SIZE_LEN;
+
+constexpr auto K2_JOB_WORKER_IGNORE_ANSWER_FLAG = static_cast<uint32_t>(1U << 0U);
```

Comment:
I think it's good idea to represent TL-functions as structures.
In the spirit of this approach, it seems better to store this constants as `private static constexpr` constants

Reply:
In general, it's a good idea, but for me it's good only in case of unification. For example, other TL-types like `string` should also have their constants defined in some namespace or class. Otherwise, we just add multiple ways to do the same thing